### PR TITLE
Added GVRImportSettings.getRecommendedMorphSettings

### PR DESCRIPTION
**Pull request by NolaDonato**
Typically the recommended assimp settings join identical vertices to increase sharing. This doesn't work for blend shapes - it will change the number of vertices in the shapes making them incompatible. This PR adds settings for importing morph shapes to avoid this sharing.

### CODE REVIEW

File: GVRf/Framework/framework/src/main/res/raw/vertex_template_multitex.vsh [hunk 1]
```diff
@@ -123,25 +127,35 @@ void main() {
...
 	viewspace_normal = vertex.viewspace_normal;
 	view_direction = vertex.view_direction;
 #ifdef HAS_MULTIVIEW
+<<<<<<< HEAD
 	 bool render_mask = (u_render_mask & (gl_ViewID_OVR + uint(1))) > uint(0) ? true : false;
      mat4 mvp = u_mvp_[gl_ViewID_OVR];
      if(!render_mask)
          mvp = mat4(0.0);  //  if render_mask is not set for particular eye, dont render that object
      gl_Position = mvp  * vertex.local_position;
+=======
```
Comment (liaxim) [inline GVRf/Framework/framework/src/main/res/raw/vertex_template_multitex.vsh]: Looks like unresolved merge conflict.

Comment (NolaDonato) [reply GVRf/Framework/framework/src/main/res/raw/vertex_template_multitex.vsh]: You are right but it does not appear in my version! I will try to fix it. Maybe I can do a force push...

**Comment by NolaDonato** [comment]
I fixed the merge conflict

**Comment by liaxim** [comment]
gvr-morph fails with this:
```
```

**Comment by liaxim** [comment]
Does gvr-morph need updates to work with this pull request?

**Pull request merged**